pytest.ini

```diff
@@ -13,5 +13,5 @@ flake8-max-line-length = 120
 # Others can be found in https://flake8.pycqa.org/en/latest/user/error-codes.html
 flake8-ignore = E402 F401 F403
 
-# --doctest-modules is used for unitest
-addopts = --doctest-modules --cov --cov-report=term-missing --flake8
+# --doctest-modules is used for unittest
```

Comment:
why modify these two lines?

Reply:
1. I think "unitest" is a spelling mistake for "unittest"
2. I don't know why only change "addopts" to "adopts" can I run pytest to test
Sorry. If these changes is not necessary, I will keep previous code of this file.

Reply:
I reviewed my code and ran it again. I think the first change could be kept, but the second change is my mistake.
I have committed again. I'm so sorry for that.

Reply:
I believe it should be ``unittes`` rather than ``unitest``